pkg/sentry/syscalls/linux/vfs2/mount.go

```diff
@@ -94,6 +93,12 @@ func Mount(t *kernel.Task, args arch.SyscallArguments) (uintptr, *kernel.Syscall
 	if flags&linux.MS_NOEXEC == linux.MS_NOEXEC {
 		opts.Flags.NoExec = true
 	}
+	if flags&linux.MS_NODEV == linux.MS_NODEV {
```

Comment:
I think it is better to do this change on a separate PR.

Reply:
We had to remove them from the `unsupportedFlags`  above as `libfuse` uses them to mount FUSE filesystems. Added a `TODO` to add support for it for other filesystems.